Allow clock pins to be distinguishable (#838)
thewca/wca-regulations: wca-guidelines.md

- 3h++) [EXAMPLE] Examples of enhancements include: new moves are possible, normal moves are impossible, more pieces or faces are visible, colors on the backside of the puzzle are visible, moves are done automatically, or the puzzles have more/different solved states.
- 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
- 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.

> - 3j++) [CLARIFICATION] Clocks with pins distinguishable from any other pin of the same side is permitted.

Reviewer: ```suggestion
- 3j++) [CLARIFICATION] On Clock, pins may be distinguishable from any other pin of the same side.
```

Reviewer: @hanwu85 don't you think this style would be easier to read and would match 3j2+?

Author: My brain was stuck and did not come up with this simple one...Thanks!
